src/Module/Photo.php

```diff
@@ -340,6 +342,7 @@ private static function getPhotoByid(int $id, $type, $customsize)
 					}
 				}
 				if (empty($mimetext)) {
+					Logger::info('Blubb');
```

Comment:
Please remove debug logging.

Reply:
Örgs ...

Reply:
You could create a `pre-commit` hook for your git repository in `.git/hooks/pre-commit` with a small bash script. I'm using it to `php -l` the commited PHP files with

```
#!/bin/bash

git diff --cached --name-status | while read st file; do
	# skip deleted files
	if [ "$st" == 'D' ]; then continue; fi
	# do a check only on the php files
	if [[ "$file" =~ ".php" ]] && ! /usr/bin/php -l $file; then
		echo "PHP syntax check failed for file: $file"
		exit 1
	fi
done
```

one could add a `grep` for "blubb" at the end so files with a "blubb" are not commited.

```
#!/bin/bash

git diff --cached --name-status | while read st file; do
	# skip deleted files
	if [ "$st" == 'D' ]; then continue; fi
	# do a check only on the php files
	if [[ "$file" =~ ".php" ]] && ! /usr/bin/php -l $file; then
		echo "PHP syntax check failed for file: $file"
		exit 1
	fi
	if [[ "$file" =~ ".php" ]] && /bin/grep "blubb" $file; then
		echo "The file $file contains a blubb"
		exit 1
	fi
done
```